clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -4698,6 +4698,19 @@ def CIR_EhTypeIdOp : CIR_Op<"eh.typeid",
   }];
 }
 
+def CIR_EhSjljSetjmp : CIR_Op<"eh.sjlj.setjmp"> {
+  let summary = "CIR setjmp operation";
+  let description = [{
+    "Dedicated setjmp operation for longjmp, a nonlocal jump"
```

Comment:
I'd like to see a more verbose description. I suggest eliminating the word "dedicated" because it doesn't really mean anything in the context of the CIR dialect. This should describe the semantics of the operation. It's OK to describe this in terms of the C `setjmp` macro and the built-in functions, but it should give at least a high-level summary of what those do.

Reply:
bringing it back because it hasn't been resolved yet